airbyte-integrations/connectors/source-github/source_github/source.py

```diff
@@ -102,18 +102,40 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
     def streams(self, config: Mapping[str, Any]) -> List[Stream]:
         authenticator = self._get_authenticator(config["access_token"])
         repositories = self._generate_repositories(config=config, authenticator=authenticator)
-        organizations = list({org.split("/")[0] for org in repositories})
+        organizations = list(set({org.split("/")[0] for org in repositories}))
```

Comment:
There is no need for this. `{org.split("/")[0] for org in repositories}` already creates `set`, which filters out duplicate elements.